aclogview/CM_Physics.cs

```diff
@@ -841,6 +841,430 @@ public enum BitfieldIndex {
+    {
+        public enum OldPublicWeenieDescPackHeader
+        {
+            PWD_Packed_None = 0,
+            PWD_Packed_PluralName = (1 << 0),
+            PWD_Packed_ItemsCapacity = (1 << 1),
+            PWD_Packed_ContainersCapacity = (1 << 2),
+            PWD_Packed_Value = (1 << 3),
+            PWD_Packed_Useability = (1 << 4),
+            PWD_Packed_UseRadius = (1 << 5),
+            PWD_Packed_Monarch = (1 << 6),
+            PWD_Packed_UIEffects = (1 << 7),
+            PWD_Packed_AmmoType = (1 << 8),
+            PWD_Packed_CombatUse = (1 << 9),
+            PWD_Packed_Structure = (1 << 10),
+            PWD_Packed_MaxStructure = (1 << 11),
+            PWD_Packed_StackSize = (1 << 12),
+            PWD_Packed_MaxStackSize = (1 << 13),
+            PWD_Packed_ContainerID = (1 << 14),
+            PWD_Packed_WielderID = (1 << 15),
+            PWD_Packed_ValidLocations = (1 << 16),
+            PWD_Packed_Location = (1 << 17),
+            PWD_Packed_Priority = (1 << 18),
+            PWD_Packed_TargetType = (1 << 19),
+            PWD_Packed_BlipColor = (1 << 20),
+            PWD_Packed_VendorClassID = (1 << 21),
+            PWD_Packed_SpellID = (1 << 22),
+            PWD_Packed_RadarEnum = (1 << 23), 
+            PWD_Packed_RadarDistance = (1 << 24),
+            PWD_Packed_HouseOwner = (1 << 25),
+            PWD_Packed_HouseRestrictions = (1 << 26),
+            PWD_Packed_PScript = (1 << 27),
+            PWD_Packed_HookType = (1 << 28),
+            PWD_Packed_HookItemTypes = (1 << 29),
+            PWD_Packed_IconOverlay = (1 << 30),
+            PWD_Packed_MaterialType = (1 << 31)
+        }
+
+        public enum BitfieldIndex
+        {
+            BF_OPENABLE = (1 << 0),
+            BF_INSCRIBABLE = (1 << 1),
+            BF_STUCK = (1 << 2),
+            BF_PLAYER = (1 << 3),
+            BF_ATTACKABLE = (1 << 4),
+            BF_PLAYER_KILLER = (1 << 5),
+            BF_HIDDEN_ADMIN = (1 << 6),
+            BF_UI_HIDDEN = (1 << 7),
+            BF_BOOK = (1 << 8),
+            BF_VENDOR = (1 << 9),
+            BF_PKSWITCH = (1 << 10),
+            BF_NPKSWITCH = (1 << 11),
+            BF_DOOR = (1 << 12),
+            BF_CORPSE = (1 << 13),
+            BF_LIFESTONE = (1 << 14),
+            BF_FOOD = (1 << 15),
+            BF_HEALER = (1 << 16),
+            BF_LOCKPICK = (1 << 17),
+            BF_PORTAL = (1 << 18),
+            // NOTE: Skip 1
+            BF_ADMIN = (1 << 20),
+            BF_FREE_PKSTATUS = (1 << 21),
+            BF_IMMUNE_CELL_RESTRICTIONS = (1 << 22),
+            BF_REQUIRES_PACKSLOT = (1 << 23),
+            BF_RETAINED = (1 << 24),
+            BF_PKLITE_PKSTATUS = (1 << 25),
+            BF_INCLUDES_SECOND_HEADER = (1 << 26),
+            BF_BINDSTONE = (1 << 27),
+            BF_VOLATILE_RARE = (1 << 28),
+            BF_WIELD_ON_USE = (1 << 29),
+            BF_WIELD_LEFT = (1 << 30),
+        }
+
+        public uint header;
+        public uint header2;
```

Comment:
Thanks. This was in there because I was so confused by the bit header until I realized I was looking for "-1" in a uint. Not having found any actual use cases for the "Old" PublicWeenieDesc, I negelected going back for a clean-up pass. So please accept my apologies for making you waste your time on that (most of your comments are relating to changes I made while trying to debug)